src/main/java/seedu/address/logic/commands/DeleteSessionCommand.java

```diff
@@ -0,0 +1,56 @@
+package seedu.address.logic.commands;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.session.Session;
+import seedu.address.model.student.Name;
+import seedu.address.model.student.Student;
+
+import static java.util.Objects.requireNonNull;
+
+public class DeleteSessionCommand extends Command {
+
+    public static final String COMMAND_WORD = "delete_session";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Deletes the session identified by the index number used in the student's list of sessions.\n"
+            + "Parameters: INDEX (must be a positive integer)\n"
+            + "Example: " + COMMAND_WORD + " John Doe" + " 1";
```

Comment:
This should evaluate to `delete_session John Doe 1` but should be `delete_session n/John Doe i/1`